akka-http-core/src/main/scala/akka/http/impl/settings/ClientConnectionSettingsImpl.scala

```diff
@@ -39,18 +38,22 @@ private[akka] final case class ClientConnectionSettingsImpl(
   require(requestHeaderSizeHint > 0, "request-size-hint must be > 0")
 
   override def productPrefix = "ClientConnectionSettings"
+
+  override def websocketRandomFactory: () ⇒ Random = websocketSettings.randomFactory
```

Comment:
deprecate/comment about keeping for bincomp?

Reply:
(deprecated on interface)